android/datastore.md

```diff
@@ -10,7 +10,7 @@ title: DataStore
 <br />
 
 **Note**
-Amplify Android is in preview mode and is not intended for production usage at this time. We welcome feedback to improve your experience in using Amplify Android.
+This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain.
```

Comment:
why not add in this line like the others? 
`To use the existing AWS Mobile SDK for Android instead, [click here.](../sdk/android/start)`

Reply:
DataStore is not present for SDKs. There is no equivalent SDK for it. Thus, did not add a note for SDK here.